apps/docs/content/components/Button/index.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Button
+description: Use Button to display the main actions on a page or form.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image showing different Button types.](https://github.com/primer/brand/assets/912236/435f104b-47a5-49d8-9568-6825f2b3789c)
+
+## Usage
+
+Buttons allow users to trigger an action or navigate to a new page when clicked. The button's label leads with an actionable verb, clearly indicating its purpose. Use the "Default" button variant primarily, and only reach for the other variants for hierarchical or functional purposes.
```

Comment:
We should mention that the button should be rendered as `<a>` not `<button>` it it's used to navigate to a new page.

Reply:
Good point! I'll add that note